.github/workflows/release-pyth-cosmwasm-contract.yml

```diff
@@ -26,6 +26,11 @@ jobs:
           npm run build-contract -- --injective
           mv ../artifacts ../injective
           zip -r injective.zip ../injective
+      - name: Build osmosis cosmwasm contract
+        run: |
+          npm run build-contract -- --osmosis
+          mv ../artifacts ../osmosis
+          zip -r injective.zip ../osmosis
```

Comment:
osmosis.zip right?

Reply:
You might need to add it to the Release step below too

Reply:
oh yeah, thanks for pointing it out